pom.xml

```diff
@@ -131,11 +131,43 @@
             <version>2.13</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>ssh-credentials</artifactId>
+          <version>1.13</version>
+          <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>mailer</artifactId>
+            <version>1.18</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-multibranch</artifactId>
+            <version>2.9.2</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.jenkinsci.plugins</groupId>
             <artifactId>pipeline-model-definition</artifactId>
-            <version>1.2.5</version>
+            <version>1.2.6</version>
```

Comment:
You could go all the way to 1.3 here, but that might require bumping other dependencies, so I defer to you.

Reply:
I agree, I just didn't want to try to sort out what the dependency impacts were here -- since it's only bumped for testing use, it's probably not critical.